src/components/TodoForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import usersFromServer from '../api/users';
+
+interface TodoFormProps {
+  onAddTodo: (title: string, userId: number) => void;
+}
+
+export const TodoForm: React.FC<TodoFormProps> = ({ onAddTodo }) => {
+  const [userSelectId, setUserSelectId] = useState(0);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [errorUserSelectId, setErrorUserSelectId] = useState(false);
+  const [errorTodoTitle, setErrorTodoTitle] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (userSelectId === 0) {
+      setErrorUserSelectId(true);
+    }
+
+    if (todoTitle === '') {
+      setErrorTodoTitle(true);
+    }
+
+    if (userSelectId === 0 || todoTitle === '') {
+      return;
```

Comment:
The check `if (userSelectId === 0 || todoTitle === '')` is redundant since these conditions are already checked individually above. You can simplify the logic by combining the error setting and return logic.